CockroachDB crashed when import data from mysql dump to a table with default value from sequence

**BUG REPORT**

1. Please supply the header (i.e. the first few lines) of your most recent
   log file **for each node in your cluster**. On most unix-based systems
   running with defaults, this boils down to the output of

     grep -F '[config]' cockroach-data/logs/cockroach.log

   When log files are not available, supply the output of `cockroach version`
   and all flags/environment variables passed to `cockroach start` instead.

```
I180620 07:21:30.063721 1 util/log/clog.go:1131  [config] file created at: 2018/06/20 07:21:30
I180620 07:21:30.063721 1 util/log/clog.go:1131  [config] running on machine: Stephen-Work
I180620 07:21:30.063721 1 util/log/clog.go:1131  [config] binary: CockroachDB CCL v2.1.0-alpha.20180604 (x86_64-apple-darwin13, built 2018/06/04 14:36:09, go1.10)
I180620 07:21:30.063721 1 util/log/clog.go:1131  [config] arguments: [cockroach start --insecure]
I180620 07:21:32.359040 23 cli/start.go:606  [config] clusterID: c94d0f5f-4adc-4436-8ed3-7c9b3ee924b5
```

Error message from `cockroach.log`
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x52aea7f]

goroutine 2153 [running]:
github.com/cockroachdb/cockroach/pkg/sql.maybeAddSequenceDependencies(0x0, 0x0, 0xc42953a300, 0xc4285f8900, 0x634d1c0, 0xc4262e0070, 0xc42918fce0, 0x0, 0x0, 0xc429aa1b18, ...)
        /go/src/github.com/cockroachdb/cockroach/pkg/sql/sequence.go:295 +0x10f
github.com/cockroachdb/cockroach/pkg/sql.MakeTableDesc(0x633f300, 0xc426290bd0, 0x0, 0x0, 0x0, 0xc42071c000, 0xc425ef5260, 0x350000019a, 0x1539cd5bb7b891ae, 0x0, ...)
        /go/src/github.com/cockroachdb/cockroach/pkg/sql/create_table.go:1060 +0x610
github.com/cockroachdb/cockroach/pkg/ccl/importccl.MakeSimpleTableDescriptor(0x633f300, 0xc426290bd0, 0xc42071c000, 0xc425ef5260, 0x350000019a, 0x1539cd5bb7b891ae, 0x4, 0xc42028ebb0, 0x1)
        /go/src/github.com/cockroachdb/cockroach/pkg/ccl/importccl/import_stmt.go:146 +0x66f
github.com/cockroachdb/cockroach/pkg/ccl/importccl.importPlanHook.func1(0x633f300, 0xc426290bd0, 0x0, 0x0, 0x0, 0xc427c60f60, 0x0, 0x0)
        /go/src/github.com/cockroachdb/cockroach/pkg/ccl/importccl/import_stmt.go:435 +0x4c6
github.com/cockroachdb/cockroach/pkg/sql.(*hookFnNode).startExec.func1(0xc425ef5200, 0x633f300, 0xc4262909c0, 0xc428935d98, 0xc428935d28)
        /go/src/github.com/cockroachdb/cockroach/pkg/sql/planhook.go:138 +0x65
created by github.com/cockroachdb/cockroach/pkg/sql.(*hookFnNode).startExec
        /go/src/github.com/cockroachdb/cockroach/pkg/sql/planhook.go:137 +0x188
```

2. Please describe the issue you observed:

- What did you do?

I want to create a table with default value get from sequence, and import data from mysql dump, the test script is like below:
```
create sequence if not exists col_id;

import table t_test (
    col_id int default nextval('col_id') not null,
    col_name string not null,
    primary key (col_id)
) MYSQLDUMP DATA ('nodelocal:///test_t_test.sql');
```

- What did you expect to see?

I expect the data can imported to the new table without crash.

- What did you see instead?

The Cockroach instance crashed.

--- a/pkg/ccl/importccl/import_stmt.go
+++ b/pkg/ccl/importccl/import_stmt.go
@@ -152,7 +152,10 @@ func MakeSimpleTableDescriptor(
 		}
 	}
 	semaCtx := tree.SemaContext{}
-	evalCtx := tree.EvalContext{CtxProvider: ctxProvider{ctx}}
+	evalCtx := tree.EvalContext{
+		CtxProvider: ctxProvider{ctx},
+		Sequence:    &importSequenceOperators{},
+	}
 	tableDesc, err := sql.MakeTableDesc(
 		ctx,
 		nil, /* txn */
@@ -172,6 +175,52 @@ func MakeSimpleTableDescriptor(
 	}
 
 	return &tableDesc, nil
+}
+
+var errSequenceOperators = errors.New("sequence operations unsupported")
+
+// Implements the tree.SequenceOperators interface.
+type importSequenceOperators struct {
+}
+
+// Implements the tree.EvalDatabase interface.
+func (so *importSequenceOperators) ParseQualifiedTableName(
+	ctx context.Context, sql string,
+) (*tree.TableName, error) {
+	return nil, errSequenceOperators
+}
+
+// Implements the tree.EvalDatabase interface.
+func (so *importSequenceOperators) ResolveTableName(ctx context.Context, tn *tree.TableName) error {
+	return errSequenceOperators
+}
+
+// Implements the tree.EvalDatabase interface.
+func (so *importSequenceOperators) LookupSchema(
+	ctx context.Context, dbName, scName string,
+) (bool, tree.SchemaMeta, error) {
+	return false, nil, errSequenceOperators
+}
+
+// Implements the tree.SequenceOperators interface.
+func (so *importSequenceOperators) IncrementSequence(
+	ctx context.Context, seqName *tree.TableName,
+) (int64, error) {
+	return 0, errSequenceOperators
+}
+
+// Implements the tree.SequenceOperators interface.
+func (so *importSequenceOperators) GetLatestValueInSessionForSequence(
+	ctx context.Context, seqName *tree.TableName,
+) (int64, error) {
+	return 0, errSequenceOperators
+}
+
+// Implements the tree.SequenceOperators interface.
+func (so *importSequenceOperators) SetSequenceValue(
+	ctx context.Context, seqName *tree.TableName, newVal int64, isCalled bool,
+) error {
+	return errSequenceOperators
 }
 
 const csvDatabaseName = "csv"

--- a/pkg/ccl/importccl/import_stmt_test.go
+++ b/pkg/ccl/importccl/import_stmt_test.go
@@ -298,6 +298,12 @@ d
 			typ:    "NOPE",
 			err:    `unsupported import format`,
 		},
+		{
+			name:   "sequences",
+			create: `i int default nextval('s')`,
+			typ:    "CSV",
+			err:    `sequence operations unsupported`,
+		},
 	}
 
 	var dataString string

